holmscan/cli.py

```diff
@@ -97,24 +219,102 @@ def run(cli_args, sub_args):
     try:
         c = Controller()
 
-        if cli_args["<command>"] == "kaboom":
-            asset = c.scan.get_asset_id(sub_args["<asset>"])
-            scan_id = c.scan.start_web_scan(
-                name="{0} Job".format(sub_args["<asset>"]),
-                schedule_is_active=False,
-                node_source_overrides=False,
-                is_was_discovery=True,
-                assets=[asset],
-                scheduled=False,
-            )
-            print(c.scan.get_web_scan(scan_id))
+        if cli_args["<command>"] == "net" and sub_args.get("asset", False):
+            data = c.scan.get_net_assets()
+            log.debug(pformat(data))
+            filtered = [[x["name"], x["uuid"]] for x in data["results"]]
+            print(tabulate(filtered, headers=["Name", "UUID"], **tabulate_args))
+        elif cli_args["<command>"] == "net" and sub_args.get("profile", False):
+            data = c.scan.get_net_profiles()
+            log.debug(pformat(data))
+            filtered = [[x["name"], x["uuid"]] for x in data]
```

Comment:
Single names variables should be avoided if possible. Add a meaning to the variable or default to the more common `item` as var name.